internal/sdkv2provider/provider.go

```diff
@@ -263,6 +263,7 @@ func New(version string) func() *schema.Provider {
 				"cloudflare_static_route":                                    resourceCloudflareStaticRoute(),
 				"cloudflare_bot_management":                                  resourceCloudflareBotManagement(),
 				"cloudflare_teams_account":                                   resourceCloudflareTeamsAccount(),
+				"cloudflare_teams_certificate":                               resourceCloudflareTeamsCertificate(),
```

Comment:
best have a chat with kenny on this one to get the naming right. we've just done a mass transition migration to get these better named and make more sense in the current landscape.

Reply:
I forgot about the rename, teams_ is replaced with zero_trust. I'll see if cloudflare_zero_trust_gateway_certificate is kosher with Kenny and Ankur

Reply:
Hi Jacob, James and Ankur have approved in the ticket, let me know if you have any other blockers on this PR.